packages/web-frontend/src/containers/MeasureLegend/labels.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
```

Comment:
It would be great to move `LabelLeft`and `LabelRight`to SpectrumLegend.js since that is the only place they are used. As a general rule it is good to have constants and components defined as close as possible to their usage.

Then this file only contains the `NoDataLabel` component and so the file can be renamed to `NoDataLabel.js`

Reply:
I decided to go one further and use LegendEntry. However, this causes some (I believe positive) change to the ui:

What do you think?
(Note: the difference in background color is not related)
## Black:

### Old:
![image](https://user-images.githubusercontent.com/32168886/96199162-8ecf4900-0fa2-11eb-8a9c-137fb6f61877.png)
![image](https://user-images.githubusercontent.com/32168886/96199182-a1e21900-0fa2-11eb-8571-a8a04c3dd92b.png)

New:
![image](https://user-images.githubusercontent.com/32168886/96198744-6266fd00-0fa1-11eb-9ffe-8da5a3cbfaa6.png)
![image](https://user-images.githubusercontent.com/32168886/96198739-5ed37600-0fa1-11eb-8d62-f738fb01ef4d.png)

## Grey:
### Old (no change on click):
![image](https://user-images.githubusercontent.com/32168886/96199211-b9b99d00-0fa2-11eb-9ca7-738dcc107302.png)

### New:
![image](https://user-images.githubusercontent.com/32168886/96198785-80ccf880-0fa1-11eb-890d-c490558226a2.png)
![image](https://user-images.githubusercontent.com/32168886/96198782-7f033500-0fa1-11eb-90bc-e64d756d9163.png)